00 Base/src/components/myComponent/myComponent.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  /*   fetchfruits,
+  deleteFruit,
+  addFruit, */
+  fetchPosts,
+  deletePost,
+  addPost,
+} from './../../redux/actions/index';
+import './styles.less';
+
+import Button from '@material-ui/core/Button';
+import Chip from '@material-ui/core/Chip';
+
+import { MyInputComponent } from '../myInputComponent/myInputComponent';
+export interface Props {
+  nameFromProps: string;
+}
+
+export const MyComponent: React.FunctionComponent<Props> = (props) => {
+  const { nameFromProps } = props;
+  const dispatch = useDispatch();
+  const posts = useSelector((state) => state.posts.posts);
+
+  return (
+    <>
+      <div className="container">
+        <h1>Hello {nameFromProps}!</h1>
+        <Button
+          variant="contained"
+          color="primary"
+          onClick={() => dispatch(fetchPosts())}
+        >
+          Get post from API
+        </Button>
+        <div className="chip-container">
+          {!!posts
+            ? posts.map((el /* : Object */) => (
```

Comment:
Please fix this commented code, you can declare an interface to type post, you can't type as Object